### PR TITLE
Describe PEP as living documentation

### PR DESCRIPTION
**Pull request by srittau**
* Say that this PEP will evolve with time.
* Remove "compliance" language, as this is an informational PEP.
  Also, if a stub or type checker would like to call itself
  "compliant" with this PEP, we would need to introduce some kind
  of versioning.
* Remove requirements that deviations must be documented.
* Try to clarify what it means that a type checker to "interpret"
  a construct.

### CODE REVIEW

File: pep-9999.rst [hunk 1]
```diff
@@ -35,21 +35,24 @@ stubs serve multiple purposes:
...
-will parse a compliant type stub without error and does not interpret any
-construct in a contradictory manner. A compliant type checker is not
-required to interpret all standard constructs, and additionally a type checker
-can support extra constructs not described in this PEP. All deviations from
-this PEP should be documented.
+A type checker that conforms to this PEP will parse a type stub only using
+constructs described here without error and does not interpret any
+construct in a contradictory manner. Type checkers are not
+required to implement checks for all these constructs, though, but
+can elect to ignore unsupported ones. Additionally a type checker
```
Comment (rchen152) [inline pep-9999.rst:51]: nit: I find the first two sentences of this paragraph a little clunky. I'd suggest rewording them as:

A type checker that conforms to this PEP will parse a type stub ~only using~ *that only uses* constructs described here without error and ~does~ *will* not interpret any construct in a contradictory manner. *However,* type checkers are not required to implement checks for all these constructs ~, though, but~ *and* can elect to ignore unsupported ones.

**Review by rchen152 — APPROVED** [review]

**Review by Solumin — APPROVED** [review]
